lms/tasks/grading.py

```diff
@@ -63,8 +64,8 @@ def sync_grade(*, lis_outcome_service_url: str, grading_sync_grade_id: int):
                     # If this is the last retry, mark this particular grade as an error
                     grading_sync_grade.success = False
                     grading_sync_grade.error_details = {"exception": str(err)}
-
                     _schedule_sync_grades_complete(grading_sync.id, countdown=1)
+                    LOG.exception("Syncing grade back to LMS failed")
```

Comment:
We were missing this for both version, if we give up on the sync log it. The log will contain the full stack trace and will be sent to sentry.